resources/darwin/bin/nerdctl

```diff
@@ -7,9 +7,14 @@ while [ -L "${scriptname}" ]; do
 done
 scriptdir="$(cd "$(dirname "${scriptname}")" && pwd)"
 
-if ! LIMA_HOME="$HOME/Library/Application Support/rancher-desktop/lima" "${scriptdir}/../lima/bin/limactl" ls --json | grep '"name":"0"' | grep -q '"status":"Running"'; then
-  echo "Rancher Desktop is not running. Please start Rancher Desktop to use nerdctl";
-  exit 1
-else
-  "${scriptdir}/rdctl" shell sudo nerdctl "$@"
-fi
+export LIMA_HOME="$HOME/Library/Application Support/rancher-desktop/lima"
+
+status=$( "${scriptdir}/../lima/bin/limactl" ls 0 -f '{{.Status}}')
```

Comment:
Doesn't this need to drop STDERR in case the VM does not exist at all?

```suggestion
status=$( "${scriptdir}/../lima/bin/limactl" ls 0 -f '{{.Status}}' 2>/dev/null)
```

Reply:
done.